### PR TITLE
Contest create UI

### PR DESCRIPTION
**Pull request by MiladS95**
### What this PR does (required):
- created ui for contest create page
- has validation to check to make sure each field is valid
- the image gallery is optional so no validation needed there 

### Screenshots / Videos (required):
- 
Screen Shot 2021-06-07 at 1 30 24 PM

### Any information needed to test this feature (required):
- added new package moment.js which will be needed to view page 

### Any issues with the current functionality (optional):
- there is a dummy array to showcase images which will be changed later to be integrated with AWS
- the handleSubmit functionality is again there to demonstrate error checking for fields will be changed later to integrate
with backend

### CODE REVIEW

File: client/src/components/DashboardNav/DashboardNav.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  to: string;
+  primary: string;
+}
+
+const DashboardNav = ({ to, primary }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Link to={to} className={classes.link}>
```
Comment (bonnieli) [inline client/src/components/DashboardNav/DashboardNav.tsx:15]: i would merge Link and Button into one if possible: https://stackoverflow.com/a/59187804/14204824

File: client/src/components/Notification/Notification.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right',
+        }}
+        open={open}
+        autoHideDuration={6000}
+        onClose={handleClose}
+        message="this is a notification"
+        action={
+          <React.Fragment>
```
Comment (bonnieli) [inline client/src/components/Notification/Notification.tsx:33]: do you need this extra Fragment wrapper?

File: client/src/pages/Contest/ContestForm/useStyles.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  root: {
+    minHeight: '100vh',
+  },
+  contestTitle: {
+    fontSize: 26,
+    textAlign: 'center',
+    margin: 30,
+    color: '#000000',
+    fontWeight: 700,
+    fontFamily: "'Poppins'",
```
Comment (bonnieli) [inline client/src/pages/Contest/ContestForm/useStyles.ts:13]: can we include Poppins as part of theme so every time you use Typography, it'll get automatically applied across the app

Comment (MiladS95) [reply client/src/pages/Contest/ContestForm/useStyles.ts]: issue is fixed now poppins was not needed here in the first place

**Pull request merged**